### ISSUE TITLE
[Request] Allow access to a color picker and a texture picker for the background visualization system

### ISSUE THREAD

**Issue by Jose-Moreno** [pencil2d/pencil#990]
### --Issue Summary--
When drawing in Pencil2D the canvas is always transparent, however in the Preferences there's a neat little section which allows you to "visualize" a given color or texture for the background. However right now this is limited to a few radial buttons.

For this proposal (which i'll update eventually) I feel it would be more appropriate to have two options.

1) A color box that when selected / clicked on displays a color picker
2) A texture / image box that when selected displays a image file selector dialog from pre-loaded textures we can ship (or bundle separately to keep the app lightweight). Mind you Pencil2D already loads two of these kind of textures, so it's not strange to let them have their own picker window.

**Comment by scribblemaniac** [comment]
I think it would be nice if we also allowed custom textures to be added to the second option, but this may cause confusion if the background does not show up in the final render.

**Comment by Jose-Moreno** [comment]
@scribblemaniac agreed that would be a great option too, to toggle if you want it to be rendered or not.
